spacy_llm/backends/rest/backend/base.py

```diff
@@ -69,31 +74,46 @@ def credentials(self) -> Dict[str, str]:
         """
 
     def retry(
-        self,
-        call_api: Callable[[], requests.Response],
+        self, call_method: Callable[..., requests.Response], url: str, **kwargs
     ) -> requests.Response:
         """Retry a call to an API if we get a non-ok status code.
         This function automatically retries a request if it catches a response
-        with an error code in `error_codes`. The amount of timeout also increases
+        with an error code in `error_codes`. The time interval also increases
```

Comment:
The exponential backoff is not actually handled by this function, I propose we move this explanation elsewhere (see other comments)

Reply:
Hm, `retry()` does the [exponential backoff though](https://github.com/explosion/spacy-llm/pull/110/files#diff-1c49ce9ad523ef937d590bdf0fc1f62143ef7b0fea500f28d34c23d190d12e38R113)?

Reply:
Ah sorry about that. I had overlooked collapsed lines, perhaps?